shared/src/main/scala/coop/rchain/store/LazyKeyValueCache.scala

```diff
@@ -25,6 +25,17 @@ class LazyKeyValueCache[F[_]: Concurrent, K, V] private[LazyKeyValueCache] (
           )
       r <- d.get
     } yield r
+
```

Comment:
I think `LazyKeyValueCache` and `LazyAdHocKeyValueCache` are totally duplicates. Do I miss something?